src/test/groovy/com/github/benmanes/gradle/versions/DependencyUpdatesSpec.groovy

```diff
@@ -627,7 +570,7 @@ final class DependencyUpdatesSpec extends Specification {
     new DependencyUpdates(project, resolutionStrategy, revision, outputFormatter, outputDir, reportfileName, checkForGradleUpdate, gradleReleaseChannel).run()
   }
 
-  private def addRepositoryTo(project) {
+  private void addRepositoryTo(project) {
```

Comment:
I had briefly moved these classes out into a helper class (and made them static), so they could be used in another spec. I later converted that spec into the OutputFormatter paradigm and no logger need the helper, so I copied them back, but I left the changes in the declarations, under the assumption that greater specificity doesn't hurt.

I later removed 'static' from `addRepositoryTo` because the `getClass` call was causing the test to fail under Java 11.